docs/awsome.md

```diff
@@ -0,0 +1,154 @@
+- dependencies: A set of name/filter pairs
+	- name: The name of the chart this chart depends upon
+	- version: A filter indicating what version of the chart is required. Example: `~1.2` (greater than or equal to 1.2.0, and less than 1.3.0)
+- maintainers: A set of maintainer names, together with an email
+- details: A single paragraph describing the chart
+
+Except for `dependencies`, all fields are required.
+
+## The README.md File
+
+The README file performs one important function: It tells the user how to use your chart. It is automatically displayed when a chart is installed.
+
+Your README.md file _may_:
+
+- Tell the user how to configure the chart (if necessary)
+- How to work with the chart once it's in Kubernetes (if this is not self-evident)
+- Explain any environment variables, secrets, annotations, or special labels
+- Provide URLs to external sources that are helpful
+
+Your REDME.md file _should not_:
+
+- Add a license constraint (all charts in `deis/charts` are Apache 2)
+- Describe the project in detail
+
+In a nutshell, the README is intended as help text for your user: It gets the user from fetching your chart to using your chart.
+
+## Manifest Files
+
+All Kubernetes manifest files should be in YAML format. We know some people prefer JSON, but we've decided for a number of reasons to adopt YAML as the single format for Helm manifests.
+
+A good simple manifest typically includes:
+
+- A replication controller (`foo-rc.yaml`)
+- A service definition for each backend service this chart relies upon (`foo-db-svc.yaml`)
+- A secrets file for each secret the RC uses (`foo-password-sec.yaml`)
+
+### Labels
+
+All Helm charts should have a `name` label and a `heritage: helm` label in their metadata sections. These provide a base-level consistency across all Helm charts. (`heritage: helm` makes it easy to search a Kubernetes cluster for all components installed via Helm.)
+
+We suggest using the following labels where appropriate:
+
+- `app`
+- `tier`
+- `provider`: Indicates that this chart provides a specific backend service (e.g. MariaDB or Percona might include `provider: mysql` since they provide MySQL compatible service)
+
+### Environment Variables
+
+We recommend documenting these in the README. Something like this is great:
+
+```
+- DB_PASSWORD: The password used to connect to the MySQL database
+```
+
+### Pods vs. Replication Controllers
+
+From the Kubernetes documentation:
+
+> [W]e recommend that you use a replication controller even if your application requires only a single pod
+
+Because replication controllers come with better lifecycle management, we suggest using RCs instead of using pods. This will give your users all of the features of a pod, but with added assurances. That said, when you use an RC to run a service that _cannot be scaled_, you should indicate this to your users either by adding the `scalable: false` label to the RC, or by saying so directly in the README.
```

Comment:
Or maybe just deploy three pod manifests so users can't scale even if they wanted to?